test/premailex/html_inline_styles_test.exs

```diff
@@ -80,7 +80,7 @@ defmodule Premailex.HTMLInlineStylesTest do
       Plug.Conn.resp(conn, 500, "{}")
     end)
 
-    parsed = Premailex.HTMLInlineStyles.process(input)
+    parsed = Premailex.HTMLInlineStyles.process(input, "style,link[rel=\"stylesheet\"][href]")
```

Comment:
This isn't necessary since that's also the fallback. If this should be tested I think it should be a second test where we only select a specific style element would work. But I don't really think you need to test this.